i18n/flows.go

```diff
@@ -211,3 +217,111 @@ func poFromExtracted(initialComment string, lang envs.Language, extracted []*ext
 
 	return po
 }
+
+// ImportIntoFlows imports translations from the given PO into the given flows
+func ImportIntoFlows(po *PO, translationsLanguage envs.Language, targets ...flows.Flow) error {
+	baseLanguage := getBaseLanguage(targets)
+	if baseLanguage == envs.NilLanguage {
+		return errors.New("can't import into flows with differing base languages")
+	} else if translationsLanguage == baseLanguage {
+		return errors.New("can't import as the flow base language")
+	}
+
+	updates := CalculateFlowUpdates(po, translationsLanguage, targets...)
+
+	applyUpdates(updates, translationsLanguage)
+
+	return nil
+}
+
+// TranslationUpdate describs a change to be made to a flow translation
+type TranslationUpdate struct {
+	textLocation
+	Base string
+	Old  string
+	New  string
+}
```

Comment:
Figuring out the list of things that should change before making the changes made it a bit easier to test this stuff.. and also gives us an easy way forward for providing a preview view to users.